libcloudforensics/providers/azure/internal/account.py

```diff
@@ -217,3 +262,213 @@ def CreateDiskFromSnapshot(
+                'to take a while.'.format(snapshot_uri))
+    copied_blob.start_copy_from_url(snapshot_uri)
+    while copied_blob.get_blob_properties().copy.status != 'success':
+      sleep(5)  # Wait for the vhd to be imported in the Azure storage container
+      logger.debug('Importing snapshot from URI {0:s}'.format(snapshot_uri))
+    logger.info('Snapshot successfully imported from URI {0:s}'.format(
+        snapshot_uri))
+
+    if not disk_name:
+      disk_name = common.GenerateDiskName(snapshot,
+                                          disk_name_prefix=disk_name_prefix)
+
+    # Create a new disk from the imported snapshot
+    creation_data = {
+        'location': region,
+        'creation_data': {
+            'source_uri': copied_blob.url,
+            'storage_account_id': storage_account_id,
+            'create_option': models.DiskCreateOption.import_enum
+        }
+    }
+
+    try:
+      logger.info('Creating disk: {0:s}'.format(disk_name))
+      request = self.compute_client.disks.create_or_update(
+          self.default_resource_group_name,
+          disk_name,
+          creation_data,
+          sku=disk_type)
+      while not request.done():
+        sleep(5)  # Wait 5 seconds before checking disk status again
+      disk = request.result()
+      logger.info('Disk {0:s} successfully created'.format(disk_name))
+    except azure_exceptions.CloudError as exception:
+      raise RuntimeError('Could not create disk from URI {0:s}: {1:s}'
+                         .format(snapshot_uri, str(exception)))
+
+    # Cleanup the temporary account storage
+    self._DeleteStorageAccount(storage_account_name)
+
+    return compute.AZDisk(self,
+                          disk.id,
+                          disk.name,
+                          disk.location,
+                          disk.zones)
+
+  def _CreateStorageAccount(self,
+                            storage_account_name: str,
+                            region: Optional[str] = None) -> Tuple[str, str]:
+    """Create a storage account and returns its ID and access key.
+
+    Args:
+      storage_account_name (str): The name for the storage account.
+      region (str): Optional. The region in which to create the storage
+          account. If not provided, it will be created in the default_region
+          associated to the AZAccount object.
+
+    Returns:
+      Tuple[str, str]: The storage account ID and its access key.
+
+    Raises:
+      ValueError: If the storage account name is invalid.
+    """
+
+    if not common.REGEX_ACCOUNT_STORAGE_NAME.match(storage_account_name):
+      raise ValueError(
+          'Storage account name {0:s} does not comply with {1:s}'.format(
+              storage_account_name, common.REGEX_ACCOUNT_STORAGE_NAME.pattern))
+
+    if not region:
+      region = self.default_region
+
+    creation_data = {
+        'location': region,
+        'sku': {
+            'name': 'Standard_RAGRS'
```

Comment:
Add a comment so we get some context on this default.
https://docs.microsoft.com/en-us/rest/api/storagerp/srp_sku_types

Reply:
Added